pkg/lsb/initscripts_test.go

```diff
@@ -5,30 +5,35 @@
 package lsb
 
 import (
-	"reflect"
+	"io"
+	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Please don't use testify, see CONTRIBUTING.md  
`The [testify](https://github.com/stretchr/testify) package should not be used.`

Reply:
Ah, that's a pitty, it makes tests so much easier to write.